ovirtlib4/vms.py

```diff
@@ -97,6 +97,21 @@ def stop_and_wait(self, state='down', *args, **kwargs):
         vm = self.get(wait_for=f"entity.status.value == '{state}'", *args, **kwargs)
         return vm[0] if vm else None
 
+    def get_management_nic(self, mng_network=defaults.MNG_NETWORK):
+        """
+        Return the management vNIC of the VM
+
+        Args:
+            mng_network (str): Management network name at the Ovirt Engine
+
+        Returns:
+            VmNic: ovirtlib VmNic class if vNIC was found, None otherwise
+        """
+        for nic in self.nics(follow="vnic_profile"):
+            if nic.entity.vnic_profile.name == mng_network:
+                return nic
+        return None
+
```

Comment:
```suggestion
        for nic in self.nics(follow="vnic_profile"):
            if nic.entity.vnic_profile.name == mng_network:
                return nic
        return None
    return [nic for nic in self.nics(follow="vnic_profile") if nic.entity.vnic_profile.name == mgmt_network]
```
I think it can be a good candidate for comprehension.

Also there needs to be some consideration to the case where there can be more than one nic using the ovirtmgmt vnic profile, so in fact this method should return some sort of Iterable

Reply:
it not the same, list comprehension return list while current algorithm returns one object
I prefer this way, it easy to understand

Reply:
But what happens in the case where the VM has multiple vnics pointing to the ovirtmgmt vnic profile? in that case, you'll simply return the first one you encounter, which might not even be the one you were looking for...